ui/page/dcrdex/dcrdex_page.go

```diff
@@ -63,24 +75,81 @@ func (pg *DEXPage) OnNavigatedTo() {
 // eventually drawn on screen.
 // Part of the load.Page interface.
 func (pg *DEXPage) Layout(gtx C) D {
+	noMultipleWalMsg := pg.isMultipleAssetTypeWalletAvailable()
+	dexDisableMsg := pg.dexEnabled()
+	overlay := layout.Stacked(func(gtx C) D { return D{} })
+	c, _ := pg.dexCore.(*dexcore.DEXCore)
+	dexSet := c != nil
```

Comment:
pg.dexCore is an Interface and not "nil" even if there's no underlying value. I didn't want to use `*dexcore.DEXCore` everywhere since we only need some core methods.